### PR TITLE
Fix syntax error, make JSHint happy

### LINKED ISSUE #22: Syntax issue in v0.6.0
Looks like a merge might have gone sideways. Looking into it.

```
$ npm t

> node-untappd@0.6.0 test
> ./node_modules/.bin/jest

 FAIL  __tests__/Request.js
  ● Test suite failed to run

    /Users/stephen/Sites/node-untappd/UntappdClient.js: Unexpected token, expected , (134:2)
        132 | 		request.end();
        133 | 		return request;
      > 134 | 	};
            | 	 ^
        135 | 
        136 | 	var hasToken = function() {
        137 | 		return !!token;

Test Suites: 1 failed, 1 total
Tests:       0 total
Snapshots:   0 total
Time:        0.677s
Ran all test suites.
```

### PR DESCRIPTION
**Pull request by stephenyeargin**
Fixes #22

A missing `)` is causing `v0.6.0` to not load correctly. This PR addresses that, as well as some whitespace cleanup and missing semicolons highlighted by JSHint.

### CODE REVIEW

File: UntappdClient.js [hunk 1]
```diff
@@ -112,25 +112,26 @@ var UntappdClient = function(debug) {
...
+			request.on("error", function() {
 				if (debug) console.log("node-untappd: error: ", arguments);
 				callback.call(that, arguments, null);
 			});
+			if(method=="POST") {
+				request.write(data);
+			}
+			request.end();
+			return request;
 		});
```
Comment (stephenyeargin) [inline UntappdClient.js:134]: This is the only line that changes functionality. The rest is cleanup.

**Comment by stephenyeargin** [comment]
@arei One more merge and we should be in good shape!

**Comment by arei** [comment]
Going to merge this, but noticed that the following block is in the wrong place, and nested inside of the `end` handler now. Still going to merge, but will fix immediately in a small patch.

**Comment by arei** [comment]
v0.6.1 is now live.

**Comment by stephenyeargin** [comment]
Thank you!

**Pull request merged**